mw-webapp/src/component/sidebar/Sidebar.tsx

```diff
@@ -24,6 +25,11 @@ interface SidebarProps {
  */
 export const Sidebar = (props: SidebarProps) => {
   const [open, setOpen] = useState(false);
+  const location = useLocation();
+
+  useEffect(() => {
+    setOpen(false);
+  }, [location.pathname]);
```

Comment:
lets close after press on navigation link, without any connections to pathname

Reply:
I think I didn't quite understand the essence of the problem

Reply:
Second option i think

Reply:
yeah, I vote for the second one. But not on any content, only on links. Is it possible?

Reply:
I think I did it))